Paco/src/com/pacoapp/paco/model/ExperimentProviderUtil.java

```diff
@@ -886,6 +906,7 @@ private Event findEventBy(String select, String[] selectionArgs, String sortOrde
       if (cursor != null) {
         cursor.close();
       }
+      readLock.unlock();
```

Comment:
Hi Bram,

Sorry for the confusion. 

When I look at the latest pull request, it looks like the unlock events are in the normal flow (not in finally blocks). Am I looking at the right pull request?

Best,
Bob

Http://www.pacoapp.com/

> On Aug 5, 2016, at 10:39 AM, Bram Bonné <notifications@github.com> wrote:
> 
> In Paco/src/com/pacoapp/paco/model/ExperimentProviderUtil.java:
> 
> > @@ -886,6 +906,7 @@ private Event findEventBy(String select, String[] selectionArgs, String sortOrde
> >        if (cursor != null) {
> >          cursor.close();
> >        }
> > +      readLock.unlock();
> I'm not sure I understand what you mean: should the unlock() not happen in the finally{} block here?
> In the new commit, this code is scrapped in favor of having locks in the insert/findEvent() methods, but the unlock()s happen in the finally{} blocks there too. Let me know if you want me to change that.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>